README.md

```diff
@@ -67,13 +62,16 @@ fn main() {
 }
 ```
 
-this will show a nice, colorful diff in your terminal, like this:
+... which has the benefit to show a nice, colorful diff in your terminal,
+like this:
 
 ```diff
 -1337
 +92
 ```
 
+More detailed information is available in the [documentation]. :-)
```

Comment:
This link doesnt work because the link ref definition starts with an uppercase D